src/components/Login.vue

```diff
@@ -36,6 +36,7 @@ async function submit() {
 		() => {
 			submitted.value = false
 			loginError.value = true
+			setTimeout(() => (loginError.value = false), 1000)
```

Comment:
what's this for?

Reply:
Removes the error message after some time